lib/drivers/docker/index.js

```diff
@@ -0,0 +1,221 @@
+var sCtl = require('strong-control-channel/client');
+var Deployment = require('./deployment');
+var EventEmitter = require('events').EventEmitter;
+var fmt = require('util').format;
+var inherits = require('util').inherits;
+var sdr = require('strong-docker-run');
+var _ = require('lodash');
+
+module.exports = exports = DockerDriver;
+
+function DockerDriver(opts) {
+  if (!opts.baseDir || !opts.console || !opts.server) {
+    throw Error('Requires baseDir, console, and server');
+  }
+  if (!(this instanceof DockerDriver)) {
+    return new DockerDriver(opts);
+  }
+  EventEmitter.call(this);
+
+  // XXX(rmg) can I put temp files here?
+  this.baseDir = opts.baseDir;
+  // XXX(rmg) for log messages? or something else?
+  this.console = opts.console;
+  // XXX(rmg) this is pm's "Server" instance, right?
```

Comment:
yes, there used to be some inconsistency in this, all `.server` in strong-pm is now a `Server` instance (at least, that's the intention, I think its like that!)